scripts/lib/emoji_utils.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+from __future__ import division, print_function
+import hashlib
+import os
+import shutil
+import sys
+
+import xml.etree.ElementTree as ET
+from six import unichr
+from PIL import Image, ImageDraw, ImageFont
+
+if False:
+    from typing import Dict, Union, Text
+
+AA_SCALE = 8
+SIZE = (136, 136)
+BIG_SIZE = tuple([x * AA_SCALE for x in SIZE])
+
+
+class MissingGlyphError(Exception):
+    pass
+
+def color_font(name, code_point, code_point_to_fname_map):
```

Comment:
We have switched to using sprite sheets from iamcal's data source and tall of these functions except `get_success_stamp()` are used to generate our old emoji farm only. So these are kind of legacy functions which we are keeping just to maintain backward compatibility and will not be used anywhere else so I think it will be good to not to move them from build_emoji.